alacritty_terminal/src/term/mod.rs

```diff
@@ -210,23 +210,29 @@ impl<'a, C> RenderableCellsIter<'a, C> {
         }
 
         let num_cols = self.grid.cols();
-        let cell = self.grid[&point];
+
+        // Since we can have non zero display offset, we must take points from a grid in buffer
+        // coordinates.
```

Comment:
```suggestion
        // Convert to absolute coordinates to adjust for the display offset.
```

Just a bit shorter and straight to the point.